src/sentry/api/bases/organization_events.py

```diff
@@ -525,6 +525,12 @@ def get_event_stats_data(
                             zerofill_results=zerofill_results,
                             dataset=dataset,
                         )
+                        # XXX: Is this done for isMetricsData somewhere else?
+                        for c in query_columns:
+                            # At least one of the columns has required extracted data
+                            if results[key][c].get("meta", {}).get("isMetricsExtractedData"):
```

Comment:
This information comes from the lines I added below:
https://github.com/getsentry/sentry/blob/b154b24ad2235d92dfec3e7294d2724dcf08d2fe/src/sentry/snuba/metrics_performance.py#L449-L451